packages/exhibitor/extra/dcos_zk_backup.py

```diff
@@ -41,12 +41,15 @@ def run_command(cmd: str, verbose: bool) -> None:
 
 def _is_zookeeper_running(verbose: bool) -> bool:
     """
-    Exhibitor and therefore ZooKeeper is in active state (running).
+    Returns whether the ZooKeeper process that Exhibitor controls is running.
     """
+    zk_pid_file = Path('/var/lib/dcos/exhibitor/zk.pid')
```

Comment:
If ZK is already stopped, does the zk.pid file still exist?

Reply:
yes